Makefile

```diff
@@ -417,6 +417,10 @@ hack-uninstall-argocd: install-helm
 hack-uninstall-cert-manager: install-helm
 	$(HELM) delete cert-manager --namespace cert-manager
 
+.PHONY: hack-ngrok
+hack-ngrok:
+	ngrok http --hostname=$(KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME) 30083
```

Comment:
Can you put this at the top of this file as `KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME ?=`

Reply:
Fixed in 8a2611cc89559318bb10986f8cee10f12cbc8044